libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5295,6 +5492,29 @@ bool GCS_MAVLINK::try_send_message(const enum ap_message id)
         send_battery2();
         break;
 
+// OpenDroneID Send Messages
+#if AP_OPENDRONEID_ENABLED
+    case MSG_OPEN_DRONE_ID_BASIC_ID:
```

Comment:
This would be better just handled all within the ODID library. It has it's own timer and it knows the mavlink channel.. no need to clutter it here. The only reason to put the sends() in here is if we want the GCS/user to be able to modify the packet stream rates... but in this case we want those fixed, right?

Reply:
Fixed, moved those functions to the OpenDrone ID class.